Gemfile

```diff
@@ -11,9 +11,7 @@ gem 'bump', require: false
 # https://github.com/lsegal/yard/pull/1296
 gem 'e2mmap'
 gem 'irb', '1.0.0'
-# Workaround for Parser 2.7.0.0.
-# It specifies the upper version until Parser 2.7.0.1 release.
-gem 'parser', '>= 2.6', '< 2.7'
+gem 'parser', '>= 2.6'
```

Comment:
Shouldn't the whole line be removed? There is no need to repeat the dependency from the gemspec here, is there?

Reply:
That's right. I should have waited for a review 💦

Reply:
:slightly_smiling_face: And since this PR depends on Parser 2.7.0.1, the dependency in the gemspec should probably be updated.

Reply:
I think it makes sense to have Parser gem 2.7.0.1 or higher for runtime. I opened the PR #7609. Thanks!